docker-compose-windows-internal.yml

```diff
@@ -40,6 +37,9 @@ x-shared-build-env: &shared-build-env
   CONFIG: ${CONFIG:-devel}
   TARGET: ${TARGET:-cobalt_install}
   NINJA_FLAGS: ${NINJA_FLAGS}
+  SCCACHE: 1
```

Comment:
Was this a bug?

Unrelated but maybe interesting: This is set again for external builds in `.github/workflows/main.yaml`

Reply:
This shouldn't affect the runtime of containers on github or buildbot. It only affects locally running containers so I have no clue if this is a bug. I just made it match what the counterpart non-internal file has for the moment.